third_party/glad/BUILD.gn

```diff
@@ -0,0 +1,17 @@
+# Copyright 2014 The Chromium Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+config("glad_public") {
+  include_dirs = [ "include" ]
+}
+
+source_set("glad") {
+  public_configs = [ ":glad_public" ]
+
+  defines = []
+
```

Comment:
I would expect at least `gl.h` to be in a `public`; without that you should be getting GN errors in your other patch.